### PR TITLE
Check sorting for unset values.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

File: tests/search/sorting/sorting_unset.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    doc.add_field('year', year) unless year.nil?
+    doc.add_field('myrank', myrank)
+    vespa.document_api_v1.put(doc)
+  end
+
+  def check_sorted(sortspec, exp_ids)
+    yql = 'select * from sources * where sddocname contains "unset"'
+    form = [['yql', yql],
+            ['hits', '10'],
+            ['presentation.format', 'json']]
```
Comment (geirst) [inline tests/search/sorting/sorting_unset.rb:33]: This should not be necessary, consider removing.

**Review by geirst — DISMISSED** [review]
👍

File: tests/search/sorting/sorting_unset.rb (hunk 1 above)
Comment (toregge) [reply tests/search/sorting/sorting_unset.rb]: Removed.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**